Utils/SettingsTableViewController.swift

```diff
@@ -15,7 +15,7 @@ class SettingsTableViewController: UITableViewController {
     static let maxAudioBitrate = "Max Audio Bitrate (bps)"
     static let maxVideoBitrate = "Max Video Bitrate (bps)"
     static let defaultStr = "Default"
-    static let codecDisclaimerText = "Set your preferred audio and video codec. Not all codecs are supported with Group rooms. The media server will fallback to OPUS or VP8 if a preferred codec is not supported."
+    static let codecDisclaimerText = "Set your preferred audio and video codec. Not all codecs are supported with Group rooms. The media server will fallback to OPUS or VP8 if a preferred codec is not supported. VP8 Simulcast should only be enabled in a Group Room."
```

Comment:
We've got quite a few disclaimers here, I wonder if we could make this clearer for customers?

One sentence that I would change.

`Not all codecs are supported with Group rooms` -> `Not all codecs are supported in Group Rooms`.

Reply:
I agree with having a cluster of disclaimers. I am open for suggestions..